components/post_view/combined_system_message/combined_system_message.test.tsx

```diff
@@ -62,15 +62,11 @@ describe('components/post_view/CombinedSystemMessage', () => {
     });
 
     test('should match snapshot, combining users removed from channel by all actors', () => {
-        const allUserIds = ['current_user_id', 'other_user_id_1', 'removed_user_id_1', 'removed_user_id_2'];
+        const allUserIds = ['current_user_id', 'other_user_id_1', 'removed_user_id_1', 'emoved_user_id_22'];
```

Comment:
I don't think this typo actually causes any problems since the snapshot change above looks fine,, but it'd probably be good to fix it to avoid future confusion